tzrec/main.py

```diff
@@ -1061,16 +1087,16 @@ def _write(
         output_cols: List[str],
     ) -> None:
         output_dict = OrderedDict()
-        for c in output_cols:
-            v = predictions[c]
-            v = v.tolist() if v.ndim > 1 else v.numpy()
-            output_dict[c] = pa.array(v)
         reserve_batch_record = reserves.get()
         if reserve_batch_record is not None:
             for k, v in zip(
                 reserve_batch_record.column_names, reserve_batch_record.columns
             ):
                 output_dict[k] = v
+        for c in output_cols:
```

Comment:
why move to here?

Reply:
the accuracy is so close, the sorting of  user_id,item_id may  be incorrect.